src/js/hyperschedule.js

```diff
@@ -48,12 +53,14 @@ const importExportSaveChangesButton = document.getElementById("import-export-sav
 const importExportCopyButton = document.getElementById("import-export-copy-button");
 
 //// Global state
+const gGreyConflictCoursesOptions = ["none", "starred", "all"];
```

Comment:
This isn't global state if it's a constant. I would suggest putting it with the other data constants.